pkg/kudoctl/cmd/install.go

```diff
@@ -36,9 +36,13 @@ func NewInstallCmd() *cobra.Command {
 	installCmd.Flags().StringVar(&vars.KubeConfigPath, "kubeconfig", "", "The file path to Kubernetes configuration file. (default \"$HOME/.kube/config\")")
 	installCmd.Flags().StringVar(&vars.GithubCredentialPath, "githubcredential", "", "The file path to GitHub credential file. (default \"$HOME/.git-credentials\")")
 	installCmd.Flags().StringVar(&vars.Instance, "instance", "", "The instance name. (default to Framework name)")
-	installCmd.Flags().StringVar(&vars.Namespace, "namespace", "default", "The namespace where the operator watches for changes. (default to")
+	installCmd.Flags().StringVar(&vars.Namespace, "namespace", "default", "The namespace where the operator watches for changes. (default to \"default\"")
 	installCmd.Flags().StringArrayVarP(&vars.Parameter, "parameter", "p", nil, "The parameter name.")
 	installCmd.Flags().StringVar(&vars.PackageVersion, "package-version", "", "A specific package version on the official GitHub repo. (default to the most recent)")
+	installCmd.Flags().StringVar(&vars.RepoName, "repo-name", "kudo-test-repo", "The storage bucket name of the repo. (default to official \"kudo-registry\")")
```

Comment:
I would propose to go the same way as git or helm goes. So there will be `kudo repo` command, by default it will come with our stable registry. Then the repo you want to use for install would be encoded in the name of the package. 

docker is the same, by default it's always pointing to docker registry, by default the packages are looked for there but you can add others.

I can work on that separately from this PR. I created #281 to track that

Reply:
> I would propose to go the same way as git or helm goes. So there will be `kudo repo` command, by default it will come with our stable registry. Then the repo you want to use for install would be encoded in the name of the package.

I like the idea, how would the encoding in the name of the package look like though?

`kubectl kudo repo add [NAME] [REPO-URL]` would do what exactly?

> 
> docker is the same, by default it's always pointing to docker registry, by default the packages are looked for there but you can add others.

Do we need a `KUDO_CONFIG` file that stores the default registry information, by default the official? Would we need also a `KUDO_TMPDIR` env var/flag?

> 
> I can work on that separately from this PR. I created #281 to track that

👍

Reply:
Yes we will need a config with repositories - probably somewhere in the home folder. Maybe let's discuss more around that in that github issue